addons/sale/models/sale.py

```diff
@@ -1619,16 +1619,21 @@ def _get_sale_order_line_multiline_description_variants(self):
 
         name = "\n"
 
+        #variants need to be displayed in the language of the receiving user
+        wanted_lang = self._context.get('lang') or 'en_US'
```

Comment:
You should use get_lang : https://github.com/odoo/odoo/blob/13.0/odoo/tools/misc.py#L1167
```suggestion
        wanted_lang = get_lang(self.env)
```